content/specs/at-uri-scheme.md

```diff
@@ -19,9 +19,9 @@ summary: A URI scheme for addressing ATP repository data.
 }
 </style>
 
-# ATP URI Scheme
+# Atproto URI Scheme
```

Comment:
I think for the URI schema specifically we should call it "AT URI Scheme". In the Lexicons, I think we call them "at-uri", and the prefix is `at://` not `atproto://`. Switching from `ATP` to `AT` in this title is helpful!

Also, just as some context, this page specifically has been linked to from an IETF request for provisional URI schema registration, so we should be particularly careful not to make large changes or move the URL of the page. The other `ATP -> atproto` changes on this page do look good to go.

Reply:
got it, helpful context thanks!